dune-project

```diff
@@ -0,0 +1,26 @@
+(lang dune 3.8)
```

Comment:
Why do you need this `dune` file at the top-level of the repository?

Reply:
But I can remove it for now, or move it to a separate PR.

Reply:
Mmh, I think that's not great: the root of hax cannot contains such a dune-project file that doesn't concern hax directly but just one of its backend.
I think you have to find a solution for the script in question.